sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -769,6 +769,44 @@ export interface BlobProperties {
     tagCount?: number;
 }
 
+// @public
+export interface BlobQueryCsvTextConfiguration extends BlobQueryTextConfiguration {
+    columnSeparator: string;
+    escapeCharacter?: string;
+    fieldQuote?: string;
+    hasHeaders: boolean;
```

Comment:
Could this be defaulted to `false`? My feeling is that generally speaking this is an optional parameter in CSV parsers.

Reply:
Let's me confirm from service feature side, if optional is acceptable, let's me make it optional with default values.

Reply:
@bterlson Make it as optional. Please refer to https://github.com/Azure/azure-sdk-for-js/pull/9484